src/mongocxx/test/collection_mocked.cpp

```diff
@@ -322,14 +322,8 @@ TEST_CASE("Collection", "[collection]") {
 
         SECTION("Fails with Options") {
             success = false;
-            expected_expire_after =
-                std::chrono::seconds(static_cast<int64_t>(std::numeric_limits<int32_t>::max()) + 1);
```

Comment:
Good catch.  That was left over from the initial change that was allowing it to go up to 64 bits before I realized the C driver wasn't going to allow that.